app/views/trainees/not_supported_route.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= content_for(:breadcrumbs) do %>
+  <%= render_component GovukComponent::BackLink.new(
+    text: 'Back',
+    href: new_trainee_path
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      Other routes not supported
+    </h1>
+
+    <p class="govuk-body">
+      Assessment Only is the only route currently supported by this service. More routes will be added soon.
+    </p>
+
+    <p class="govuk-body">You can:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        <%= govuk_link_to "go back and change your answer", new_trainee_path  %>
+      </li>
+      <li>
+        <a class="govuk-link" href="#" style="color:red; font-weight: bolder">use DTTP to register a different route</a>
```

Comment:
inline styles?

Reply:
Not ideal, but it's only to help us identify that we need the DTTP url to be added

Reply:
@aliuk2012 you could add css to style all links with `#` hrefs? or have this pull from a translation file and make sure that gets updated?

Reply:
The CSS for `#` hrefs is a good shout I think.